server/chat-plugins/abuse-monitor.ts

```diff
@@ -69,11 +75,17 @@ export const settings: FilterSettings = (() => {
 	}
 })();
 
+interface PunishmentSettings {
+	count?: number;
+	punishment: string;
```

Comment:
```suggestion
	punishment: typeof PUNISHMENTS[number];
```
I think this will work?

Reply:
Why was this resolved?

Reply:
Dunno. Fixed, though.